store/sqlstore/post_store.go

```diff
@@ -197,13 +197,12 @@ func (s *SqlPostStore) GetFlaggedPosts(userId string, offset int, limit int) sto
 	})
 }
 
-func (s *SqlPostStore) GetFlaggedPostsForTeam(userId, teamId string, offset int, limit int) store.StoreChannel {
-	return store.Do(func(result *store.StoreResult) {
-		pl := model.NewPostList()
+func (s *SqlPostStore) GetFlaggedPostsForTeam(userId, teamId string, offset int, limit int) (*model.PostList, *model.AppError) {
+	pl := model.NewPostList()
 
-		var posts []*model.Post
+	var posts []*model.Post
 
-		query := `
+	query := `
             SELECT
```

Comment:
Please restore the old indentation of the query.

Reply:
Actually, I think the new indentation makes mores sense, so... I would leave it as is now.